### PR TITLE
feat: Adding UITests harness for extensions

### PR DESCRIPTION
**Pull request by nickrandolph**
GitHub Issue: #511

### CODE REVIEW

File: src/Uno.Extensions.Navigation.UI/Region.cs [hunk 1]
```diff
@@ -48,6 +48,7 @@ public static class Region
 
 	private static void AttachedChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
 	{
+		Console.WriteLine("******** Attached");
```
Comment (jeromelaban) [inline src/Uno.Extensions.Navigation.UI/Region.cs:51]: Remove?

**Review by jeromelaban — COMMENTED** [review]
Will you be integrating the CI tests in a different PR? In all cases, you can use https://github.com/unoplatform/uno.todo/pull/258 as the base.

File: testing/TestHarness/.vsconfig [hunk 2]
```diff
@@ -0,0 +1,94 @@
+{
+  "version": "1.0",
```
[the remark this answers is not in the document]
Comment (nickrandolph) [reply testing/TestHarness/.vsconfig:2]: Will update based on outcome of this discussion: https://github.com/unoplatform/uno.extensions/issues/515

**Comment by nickrandolph** [comment]
> Will you be integrating the CI tests in a different PR? In all cases, you can use [unoplatform/uno.todo#258](https://github.com/unoplatform/uno.todo/pull/258) as the base.

Yes, I'll add to CI in subsequent PR

**Review by francoistanguay — APPROVED** [review]

**Pull request merged**